vipps-order-management-api.md

```diff
@@ -244,14 +244,14 @@ Example request:
 
 Headers:
 
-```text
+```json
```

Comment:
Forgot this one? 😄

Reply:
Oh I didn't know you meant everywhere! :-D

Reply:
@flinkgutt Ok, so it's everywhere you use headers or get a response?

Reply:
I may need to look back over the other repos. I'll add a note and be on the look out. Any more helpful ways to tell if it's http vs. shell/bash vs json?  There are many that have both http and json, so I used http on those. But also some that I've just used "text" because it looks like an output with lots of gibberish. Maybe those should be http.

Reply:
I've just tested out how the formatting looks in docusaurus and it looks much nicer with json than http.
![formatting](https://user-images.githubusercontent.com/104833234/191676980-0cfb95c9-2523-46a4-80d1-5395ba4ae408.png)

Reply:
But for headers, maybe it's better as http.

Reply:
Yes, my comment was just for the HTTP Headers. :)
But very good that you checked it out how it actually looks! 👍